src/cache_cf.cc

```diff
@@ -2402,8 +2402,10 @@ parse_peer(CachePeers **peers)
         p->connect_fail_limit = 10;
 
 #if USE_CACHE_DIGESTS
-    if (!p->options.no_digest)
-        p->digest = new PeerDigest(p);
+    if (!p->options.no_digest) {
+        const auto pd = new PeerDigest(p);
+        p->digest = cbdataReference(pd);
```

Comment:
> PR description: "accidentally lost another digest lock while converting PeerDigest::peer to CbcPointer"

This code is not really about PeerDigest::peer. It is about CachePeer::digest, which has not been (and should not be) converted to CbcPointer. If you agree, please replace "while converting PeerDigest::peer to CbcPointer" with "while inlining peerDigestCreate()" or something like that.

Reply:
Ok.